lib/ProductOpener/Packaging.pm

```diff
@@ -505,6 +505,15 @@ sub apply_rules_to_augment_packaging_component_data ($product_ref, $packaging_re
 		$packaging_ref->{"shape"} = "en:coffee-capsule";
 	}
 
+	# If the shape is bottle and the material is glass, mark recycling as recycle
+	if (    (defined $packaging_ref->{"shape"})
+		and ($packaging_ref->{"shape"} eq "en:bottle")
+		and (defined $packaging_ref->{"material"})
+		and ($packaging_ref->{"material"} eq "en:glass"))
```

Comment:
We should add a condition to check that we don't already have a value for $packaging_ref->{"recycling"}

If we have a value, we should not override it. Some glass bottles are not to be recycled, but to be returned to the store for instance.